enhancements/update/update-blocker-lifecycle/README.md

```diff
@@ -0,0 +1,190 @@
+---
+title: update-blocker-lifecycle
+authors:
+  - "@wking"
+reviewers:
+  - "@LalatenduMohanty"
+approvers:
+  - TBD
+creation-date: 2020-09-11
+last-updated: 2021-06-17
+status: implementable
+---
+
+# Update-blocker Lifecycle
+
+We occasionally have bugs which impact update success or the stability of the target release.
+When that happens, we protect users by [removing the impacted updates from our update recommendations][graph-data-block].
+This enhancement describes the process used to identify these bugs and determine which edges should be blocked because of them.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The lifecycle for update blockers looks like:
+
+<div style="text-align:center">
+  <img src="flow.svg" width="100%" />
+</div>
+
+Currently all tracking through the lifecycle is manual, and it is tedious for graph-admins to audit bugs with `UpgradeBlocker` to see where they are in the lifecycle and, when necessary, poke component teams about outstanding impact statement requests.
+Having an explicit, machine-readable lifecycle reduces the chances that suspected update blockers fall through the cracks by clarifying the responsible parties for moving the bug to the next stage, which supports tracking and automated reminders.
+
+With the changes from this enhancement, the queues become:
+
+* [Suspect queue][suspect-queue].
+* [Component developer queue][component-dev-queue] (individual component teams probably want to add additional filtering for their components).
+* [Graph-admin queue][graph-admin-queue].
+
+## Motivation
+
+### Goals
+
+* Clearly define, in a machine-readable fashion, the currently responsible party for bugs in the update-blocker lifecycle.
+
+### Non-Goals
+
+This enhancement does not attempt to:
+
+* Cover issues which have not yet arrived in Bugzilla.
+* Cover bugs which do not have the `UpgradeBlocker` keyword.
+    For example, bugs with just the `Upgrades` keyword are not included in the update-blocker lifecycle.
+* Remove the `UpgradeBlocker` keyword, because that might disrupt existing consumers.
+
+## Proposal
+
+Add new `Whiteboard` labels for `ImpactStatementRequested`, `ImpactStatementProposed`, and `UpdateRecommendationsBlocked`.
+Write tooling that automatically:
+
+* Removes the new labels if any labels from later in the process are set.
+* Adds `Upgrades` and `UpgradeBlocker` to any bugs with any of the new labels.
```

Comment:
@wking have you brought this use case to the jira migration team to make sure they are thinking about how to support things like this when we go to jira?